lib/tasks/db.rake

```diff
@@ -59,7 +59,7 @@ namespace :db do
   task :reseed => [:clear, 'db:migrate', 'db:seed']
 
   desc 'CCCD task: clear the database, run migrations, seeds and reloads demo data'
-  task :reload do
+  task :reload => :environment do
```

Comment:
```suggestion
  task reload: :environment do
```

Not a blocker (considering that the rockets syntax is used everywhere else in the tasks).